mozilla-release/mobile/android/base/java/org/mozilla/gecko/controlcenter/OverviewFragment.java

```diff
@@ -357,7 +357,7 @@ public void refreshUI() {
 
     @Override
     public void onClick(View v) {
-        if (controlCenterSettingsData == null) {
+        if (controlCenterSettingsData == null || controlCenterSettingsData.getBundle("data") == null) {
```

Comment:
I would prefer to use || GeckoBundleUtils.safeGetBundle(controlCenterSettingsData,"data") == null )

Reply:
It will never be null if you use `safeGetBundle`.

Reply:
Best would be to replace all the `get` calls to `safeGet`. There are still some palces in the code where we didnt change to `safeGet`. That would prevent any crash which is happening.